dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -44,7 +41,7 @@ JWT_ENDPOINT_FILTER_MATCH_NOT_FOUND.useraction=Verify that the URL path points t
 
 # Do not translate "JSON Web Token", "JWT"
 # 0=Config ID of JWT provider
-JWT_CONFIG_SERVICE_NOT_AVAILABLE=CWWKS6005E: The JSON Web Token (JWT) configuration service is not available for provider [{0}].
+JWT_CONFIG_SERVICE_NOT_AVAILABLE=CWWKS6005E: The JSON Web Token configuration service is not available for provider [{0}].
```

Comment:
did you mean to the `(JWT)` out?

Reply:
Yep. This was actually the primary change I wanted to get in because this message gets included as the reason text in a 404 response and the parentheses were being encoded unexpectedly. I figured the "(JWT)" part wasn't needed for that anyway. I'd also argue we need to reconsider whether an NLS message should be getting put into an `HttpServletResponse.sendError(int, String)` call in the first place.